tests/test_git.py

```diff
@@ -301,3 +302,95 @@ def test_file_exists_in_repo(
     )
 
     assert result == expected_result
+
+
+@pytest.mark.parametrize(
```

Comment:
this looks redundant, to some extent with the `test_data` file - just checking if this is intentional.

Reply:
this mark could probably also benefit from some light commenting

Reply:
Thanks, yes, some of these are getting a bit redundant and might have an opportunity to be reused instead of redeclared. It's been hard to predict what might be needed within each function however so I'm hesitant to "template" these or make them static. I'll add in a comment for now here to help clarify what the mark and its contents are about.